COM-465 - Horizontal form layout (#1755)
vivid-planet/comet: .changeset/khaki-eels-try.md

---
"@comet/admin": minor
---

> `useObservedWidth` from `src/stack/breadcrumbs/utils` is moved to its own file `src/util/useObservedWidth`

Reviewer: This is an internal change which isn't needed in the changelog

> The prop `horizontal` for `FieldContainer` only takes effect, when it exceeds `600px` in width

Reviewer: ```suggestion
Only use horizontal layout in `FieldContainer` when it exceeds `600px` in width
```
